assets/js/components/NewMetricBadge.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * NewMetricBadge component.
+ *
+ * Site Kit by Google, Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { Fragment } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { Tooltip } from 'googlesitekit-components';
+import Badge from './Badge';
+import Link from './Link';
+
+function NewMetricBadge( { tooltipTitle, learnMoreLink } ) {
+	return (
+		<Tooltip
+			tooltipClassName="googlesitekit-new-metric-badge__tooltip"
+			title={
+				<Fragment>
+					<p>{ tooltipTitle }</p>
```

Comment:
Using a paragraph here adds margin that isn't in the design. I think we can perhaps even omit the wrapping element here altogether, and simply add a line break so that the learn more link is always on a new line, but without added space between, as in the design.